src/core/text/Text.js

```diff
@@ -373,6 +375,38 @@ Text.prototype.updateText = function ()
 };
 
 /**
+ * Render the text with letter-spacing.
+ *
+ * @private
+ */
+Text.prototype.drawLetterSpacing = function(text, x, y, isStroke)
+{
+    var style = this._style;
+
+    // letterSpacing of 0 means normal
+    var letterSpacing = style.letterSpacing;
+
```

Comment:
nice job man! I have one suggestion, how about if letterSpacing is 0, we avoid looping through all the characters and use fillText / StrokeText directly? Otherwise all good man!